canonical/ubuntu.com
# WD-9618 - update core home page

## Done

Update core home page according to figma and copy doc

## QA

- [demo link](https://ubuntu-com-13754.demos.haus/core)
- [copy doc](https://docs.google.com/document/d/1UFpefDRSAcndNBnWp0hDyzqk74aciqhBWJEi05htWmU/edit)
- [figma](https://www.figma.com/file/5kDdBtPRqthyPz263YV91U/24.04-Ubuntu-Core-rebrand?type=design&node-id=711-3453&mode=design&t=3k9mbyENz2DBU0hh-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to the demo and copy doc

## Issue / Card
[WD-9618](https://warthogs.atlassian.net/browse/WD-9618)

[WD-9618]: https://warthogs.atlassian.net/browse/WD-9618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
